tests/common_fixtures.py

```diff
@@ -1,38 +1,66 @@
 from http import HTTPStatus
+from os import environ
 from typing import Callable, Generator
 from unittest.mock import Mock
 
 import pytest
 import requests
 import v3io.dataplane
 
+import mlrun.api.utils.singletons.db
+import mlrun.api.utils.singletons.projects_manager
+import mlrun.config
+import mlrun.utils
 from mlrun.api.db.sqldb.db import SQLDB
-from tests.conftest import init_sqldb
+from mlrun.api.db.sqldb.session import create_session, _init_engine
+from mlrun.api.initial_data import init_data
+from mlrun.api.utils.singletons.db import initialize_db
+from mlrun.config import config
+from tests.conftest import root_path, rundb_path, logs_path
 
 session_maker: Callable
 
 
+@pytest.fixture(autouse=True)
+# if we'll just call it config it may be overridden by other fixtures with the same name
+def config_do_not_override():
+    environ["PYTHONPATH"] = root_path
+    environ["MLRUN_DBPATH"] = rundb_path
+    environ["MLRUN_httpdb__dirpath"] = rundb_path
+    environ["MLRUN_httpdb__logs_path"] = logs_path
+    environ["MLRUN_httpdb__projects__periodic_sync_interval"] = "0 seconds"
+    log_level = "DEBUG"
+    environ["MLRUN_log_level"] = log_level
+    # reload config so that values overridden by tests won't pass to other tests
+    mlrun.config.config.reload()
+
+
 @pytest.fixture
 def db():
     global session_maker
     dsn = "sqlite:///:memory:?check_same_thread=false"
     db_session = None
     try:
-        session_maker = init_sqldb(dsn)
-        db_session = session_maker()
+        config.httpdb.dsn = dsn
+        _init_engine(dsn)
+        init_data()
+        initialize_db()
+        db_session = create_session()
         db = SQLDB(dsn)
         db.initialize(db_session)
     finally:
         if db_session is not None:
             db_session.close()
+    mlrun.api.utils.singletons.db.initialize_db(db)
```

Comment:
`init_data()`
`initialize_db()`
`mlrun.api.utils.singletons.db.initialize_db(db)`
All "look" like they do the same thing or very similar things (data is slightly different but it's not clear exactly what it does).

Maybe a slight rename of some, or namespacing with the module (like the last one) can help make the individual functionality clearer to the reader?

Reply:
There's already a TODO on that

Reply:
🤷